File: Day2-Operators/Javascript/main.js
'use strict';

var _input = '';
var _index = 0;
process.stdin.on('data', (data) => { _input += data; });
process.stdin.on('end', () => {
	_input = _input.split(new RegExp('[\n ]+'));
	main(+(_input[0]), +(_input[1]), +(_input[2]));    
});

function main(mealCost, tipPercent, taxPercent) {
    let tip = mealCost * tipPercent / 100;
    let tax = mealCost * taxPercent / 100;
        
    let totalCost = Math.round(mealCost + tax + tip); 

    console.log("The total meal cost is " + totalCost + " dollars.");
}

File: Day2-Operators/C/main.c
#include <stdio.h>
#include <math.h>

int main() {
    double mealCost;
    int tipPercent;
    int taxPercent;

    scanf("%lf", &mealCost);
    scanf("%d", &tipPercent);
    scanf("%d", &taxPercent);
    
    double tip = mealCost * tipPercent / 100;
    double tax = mealCost * taxPercent / 100;
    int totalCost = (int) round(mealCost + tip + tax);
    
    printf("The total meal cost is %d dollars.", totalCost);
    
    return 0;
}